.github/workflows/metadata-ingestion.yml

```diff
@@ -69,6 +69,17 @@ jobs:
             **/build/reports/tests/test/**
             **/build/test-results/test/**
             **/junit.*.xml
+      - name: Upload coverage to Codecov
+        if: ${{ always() && matrix.python-version == '3.10' && matrix.command != 'lint' }}
```

Comment:
can we add a `github.repository == 'acryldata/datahub'` condition here?

Reply:
This has 
```
fail_ci_if_error: false
```
So this won't fail if the upload does not work.